aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -70,8 +70,8 @@ def define(cls, spec):
         spec.input('options', valid_type=orm.Dict, required=False,
             help='The metadata options that are to be passed to the calculation job.')
         spec.input('automatic_parallelization', valid_type=orm.Dict, required=False,
-            help='When specified, the work chain will first launch an initialization calculation to determine the '
-                 'dimensions of the problem and based on this will try to set optimal parallelization flags.')
+            help='When True, the work chain will first launch an initialization calculation to determine the '
```

Comment:
> When True

This is not correct, because the node type is not a `Bool` but a `Dict`. So the switching will happen if a value is passed at or not, not its content. I will fix this and then amend your changes in my original commit. Since you added a commit, I think that if I were to squash merge, the resulting commit would be attributed to you instead of me. In the future then I think it is better to just request these changes and the PR owner will have to apply them

Reply:
Ah, sorry, my bad.
OK, just wanted to save you some time.

Reply:
No worries, appreciate the gesture in any case.